future-apps/payroll/app/src/screens/components/PreviousSalary.js

```diff
@@ -107,8 +107,25 @@ const StyledFilters = styled.div`
   }
 `
 
+function getExchangedAmount (payment) {
+  return (payment.amount.amount / payment.exchangeRate.amount)
```

Comment:
Wouldn't be more clear if we had just `payment.exchangeRate`?

Reply:
Yes, could be but We dont know if they want to show or if we need to use exchangeRate (as is right now) for another calculation or to show it